### ISSUE TITLE
007: Everything or Nothing bad performance 

### ISSUE THREAD

**Issue by ghost** [xemu-project/xemu#1003]
### Title

https://xemu.app/titles/45410042/#007-Everything-or-Nothing

### Bug Description

Game runs very bad, framerate drops as low as 3fps even at 1x resolution scaling.
![xemu exe Screenshot 2022 06 01 - 00 10 48 81](https://user-images.githubusercontent.com/103341098/171298832-89db7ffd-b849-4feb-9b45-b51273945867.png)
![xemu exe Screenshot 2022 06 01 - 00 22 14 24](https://user-images.githubusercontent.com/103341098/171299915-723051f4-7d6d-4509-867c-e367f4f634ae.png)
![xemu exe Screenshot 2022 06 01 - 00 22 44 55](https://user-images.githubusercontent.com/103341098/171299928-336a7810-cdb3-4097-b5e5-c8018fbe7da8.png)

### Expected Behavior

The framerate should not be this low.

### xemu Version

0.7.25

### System Information

Windows 10
Ryzen 5 3600x
RTX 2060
NVIDIA Driver: 30.0.15.1215

**Comment by HadetTheUndying** [comment]
Might be a duplicate of https://github.com/mborgerson/xemu/issues/278 I can test here in a few. I think I own the game, just need to dig it out.

**Comment by HadetTheUndying** [comment]
Yes this is a duplicate of that issue, could you leave a comment saying this game is effected too? I just did some performance profiling and it has the same exact problem as the other games referenced

**Comment by Triticum0** [comment]
Tested on verison 0.8.49 and there is no change on both vulkan or opengl
![Image](https://github.com/user-attachments/assets/c04896a1-a6fa-4d0e-a239-776ae998f8af)

**Comment by kazmerb** [comment]
Can confirm, running on a ridiculously overpower rig I get the same performance.
To compare, I was just playing PsyOps: The Mindgate Conspiracy with no chop or performance hits.
Big, big disappointment as the only chance I ever got to play this as a kid was on a demo disc.
